### DISCUSSION TITLE
Can you make the documentation of react-native-edge-to-edge more comprehensive? It may not be very friendly to beginners.

### DISCUSSION THREAD

**Discussion by scapking** [zoontek/react-native-edge-to-edge#55] [Q&A] [answered]
react-native-edge-to-edge has been officially recognized. I think react-native-edge-to-edge has great potential and is very powerful.

Although Android 15 and above use immersive display by default, there are still a large number of devices below Android 15 in use.

For example, from creating an expo app to configuring react-native-edge-to-edge and code, as well as actual effects.

If this is possible, your Github repository will attract more beginners to like and collect it.

Then you will be recognized, and perhaps you can also receive sponsorship from some more authoritative organizations, so it is very necessary to have more comprehensive documents.

**Comment by zoontek** [comment]
Sure, what did you not understand?

**Reply by scapking** [reply]
I don't understand it at all, and no matter what I try, it doesn't really work. You can laugh at me, but can you help me solve it? If you can

**Reply by scapking** [reply]
Because the currency is different, it is also troublesome to exchange, otherwise I will try to buy you a cup of coffee

**Reply by zoontek** [reply, marked as answer]
1. You add it to your project using `npm i -S react-native-edge-to-edge`, or any package manager you want.
2. You add the expo plugin by adding it to your `app.json` (`"plugins": ["react-native-edge-to-edge"]`)
3. You [create a build](https://docs.expo.dev/develop/development-builds/create-a-build/): `npx expo prebuild --clean && npx expo run:android`
4. Remove `expo-status-bar`, `expo-navigation-bar`, etc. Replace them with `SystemBars`

By default, edge-to-edge is semi opaque with buttons navigation, transparent with gesture navigation ([read this](https://github.com/zoontek/react-native-edge-to-edge/issues/53#issuecomment-2554459214))

**Reply by zoontek** [reply]
I updated the README, it should be easier to understand now.
